web/locales/cs.yml

```diff
@@ -0,0 +1,6 @@
+cs:
+  RecurringJobs: Pravidelně opakované
```

Comment:
but I Sidekiq original users camel case, I'd preffer to hold their standards to have some strings (e.g. 'Name') translated from Sidekiq original yaml